apps/notifications/src/panel/templates/comment-reply-input.jsx

```diff
@@ -140,7 +137,7 @@ const CommentReplyInput = createReactClass( {
 
 		// Reset the field if there's no valid user input
 		// The regex strips whitespace
-		if ( '' == this.state.value.replace( /^\s+|\s+$/g, '' ) ) {
+		if ( '' === this.state.value.replace( /^\s+|\s+$/g, '' ) ) {
```

Comment:
`value.trim()` should do exactly the same thing: remove whitespace (including tabs, newlines etc) from both ends.